testsuite/features/secondary/allcli_action_chain.feature

```diff
@@ -9,38 +9,23 @@ Feature: Action chains on several systems at once
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
-  Scenario: Pre-requisite: Use the Test-Channel-x86_64 in Red Hat-like minion
-    Given I am on the Systems overview page of this "rhlike_minion"
-    When I follow "Software" in the content area
-    And I follow "Software Channels" in the content area
-    And I wait until I do not see "Loading..." text
-    And I check radio button "Test-Channel-x86_64"
-    And I wait until I see "Test-Channel-x86_64 Child Channel" text
-    And I uncheck "Test-Channel-x86_64 Child Channel"
-    And I click on "Next"
-    Then I should see a "Confirm Software Channel Change" text
-    When I click on "Confirm"
-    Then I should see a "Changing the channels has been scheduled." text
-    When I follow "scheduled" in the content area
-    And I wait until I see "1 system successfully completed this action." text, refreshing the page
-
   Scenario: Pre-requisite: downgrade packages before action chain test on several systems
     When I enable repository "test_repo_rpm_pool" on this "sle_minion"
-    And I enable repository "test_repo_rpm_pool" on this "rhlike_minion"
-    And I remove package "andromeda-dummy" from this "rhlike_minion" without error control
+    And I enable repository "test_repo_rpm_pool" on this "ssh_minion"
+    And I remove package "andromeda-dummy" from this "ssh_minion" without error control
     And I remove package "andromeda-dummy" from this "sle_minion" without error control
     And I install package "andromeda-dummy-1.0" on this "sle_minion"
-    And I install package "andromeda-dummy-1.0" on this "rhlike_minion"
+    And I install package "andromeda-dummy-1.0" on this "ssh_minion"
     And I refresh the metadata for "sle_minion"
-    And I refresh the metadata for "rhlike_minion"
+    And I refresh the metadata for "ssh_minion"
```

Comment:
Why are you changing here a RH-like client by a SLES SSH client?
This seems out of the scope of this PR.

If you want this change, let's do it in a different PR and we can discuss it there.

Reply:
That is because they need to have the same base-channel in this feature
rhlike_minion does not have the SLE base channel so this test cannot pass with it.

Reply:
OK, I see what you mean. What a limitation! OK, let's change it then.
@mcalmer I think this could be an interesting scenario for your awareness.